contracts/interfaces/IVaultManager.sol

```diff
@@ -5,4 +5,5 @@ interface IVaultManager{
     function depositByToken(address _tokenAddress, address from, uint _amount) external returns (uint vaultId);
     function withdrawByVaultId(uint _vaultId, address to, uint _amount) external;
     function vaultIdToTokenAddress(uint _vaultId) external view returns (address token);
+    function tokenToVaultId(address _tokenAddress) external view returns (uint vaultId);
```

Comment:
I think we need to use it only after depositByToken
and he returns the same value. let's copy that value.
we got N vault for each. this return the just last,
we need to take the specific from other deposit, then we need to set the other to 0

Reply:
It's worth mentioning that `tokenToVaultId` is being changed to `tokenToVaultIds` here https://github.com/The-Poolz/VaultManager/pull/22. Now it will hold all the N vaults, where the latest vault will be considered as the current vault and will be used for deposits.

Reply:
Don't think we need it in the interface